src/bref/src/SymfonyRequestBridge.php

```diff
@@ -0,0 +1,151 @@
+<?php
+
+namespace Runtime\Bref;
+
+use Bref\Context\Context;
+use Bref\Event\Http\HttpRequestEvent;
+use Bref\Event\Http\HttpResponse;
+use Riverline\MultiPartParser\StreamedPart;
+use Symfony\Component\HttpFoundation\File\UploadedFile;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * Bridges Symfony requests and responses with API Gateway or ALB event/response
+ * formats.
+ *
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ *
+ * @internal
+ */
+class SymfonyRequestBridge
+{
+    public static function convertRequest(HttpRequestEvent $event, Context $context): Request
+    {
+        Request::setTrustedProxies(['127.0.0.1'], Request::HEADER_X_FORWARDED_FOR | Request::HEADER_X_FORWARDED_HOST | Request::HEADER_X_FORWARDED_PORT | Request::HEADER_X_FORWARDED_PROTO);
```

Comment:
Why is this needed? Can't we just set the real headers, and not have the forwarded for headers making an appearance?

Reply:
Yeah, We could. But Im not sure what all sideeffects would be. Think if multiple proxies etc. It is probably fine, but I wasn't 100% sure so I left it for a future update.